src/sass/modules/_m-vet-nav.scss

```diff
@@ -1,11 +1,15 @@
+// The distance from the top of the sceen that the mobile navigation menu should appear
+$height-offset: 111.16px;
+
 body.va-pos-fixed {
   width: 100%;
   height: 100%;
 }
 
 #vetnav {
   background-color: $color-primary-darkest;
-  height: 100%;
+  height: calc(100% - #{$height-offset});
+  top: $height-offset;
```

Comment:
Is this necessary? From a quick test on the Heroku instance, removing `top` didn't seem to affect the menu's layout.

Reply:
Now that you mention it I guess it's not necessary since the `$height-offset` is so close to being exactly equal to the default location the navigation menu appears. I kinda think it makes sense to be there for clarity though. What do you think?

Reply:
Doesn't hurt to have it, I suppose. It can be clearer to specify both a `top`/`bottom` and `left`/`right`.